financial-templates-lib/clients/ExpiringMultiPartyClient.js

```diff
@@ -51,7 +56,7 @@ class ExpiringMultiPartyClient {
   // Whether the given undisputed `liquidation` (`getUndisputedLiquidations` returns an array of `liquidation`s) is disputable.
   // `tokenRedemptionValue` should be the redemption value at `liquidation.time`.
   isDisputable = (liquidation, tokenRedemptionValue) => {
-    return !this._isUnderCollateralized(liquidation.numTokens, liquidation.amountCollateral, tokenRedemptionValue);
+    return !this._isUnderCollateralized(liquidation.numTokens, liquidation.amountCollateral, "0", tokenRedemptionValue);
```

Comment:
one downside with adding the parameter to the `_isUnderCollateralized` function is that the `withdrawalRequest` is always zero for a liquidation. An alternative implementation would be to make the call to `this._isUnderCollateralized` on line 34 be this instead:

```
this._isUnderCollateralized(
        position.numTokens,
        toBN(position.amountCollateral).sub(toBN(position.withdrawalRequestAmount)).toString(),
        tokenRedemptionValue
      )
```

This has the same effect but does not change the `_isUnderCollateralized` function. Thoughts on these options?

Reply:
+1 this is actually very confusing. I think your suggested approach is better because it makes more sense for `_isUnderCollateralized` to return simply if the a given collateral ratio is too low. Whether that collateral include requested withdrawal amounts or not should be left to the caller. Also, if you need to pass in a "0" in many cases, then you might as well make that change upstream.

Nevertheless, I would add a comment in `isDisputable` that the way `liquidation.amountOfCollateral` is computed in the first place is "net of requested withdrawal amounts". This way the user knows that the collateral in question includes requested withdrawals.

Reply:
ok cool. I'll implement the change.

Reply:
Agreed. I like the second approach better.